src/containers/EMS/ems.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+//import logo from './logo.svg';
+import '../../App.css';
```

Comment:
I don't think you need to import the App.css file again. As long a stylesheet is imported once somewhere in reachable code, it will be bundled by webpack into the final SPA payload...

Reply:
In this case, App.css is imported in the main App.js file on line 3. If we follow a convention of only importing a stylesheet in the corresponding javascript file with the same name, it will be easier to keep track of (i.e. we don't end up down a "where did this style come from?!" chase if a file is imported and referenced multiple places).

Reply:
This all sounds great! Thanks for the feedback Francis. I have do not have experience with sass but I have heard about how useful it can be. I will do some research on and incorporate it.